examples/functions-with-defaultable-parameters/functions_with_defaultable_parameters.bal

```diff
@@ -16,24 +16,36 @@ function printSalaryDetails(int baseSalary,
 }
 
 public function main() {
-    // Calls the function by passing only the `baseSalary`.
-    // The `annualIncrement` and the `bonusRate` defaults to 20 and 0.02
+    // Call the function by passing only the `baseSalary`.
+    // The `annualIncrement` and the `bonusRate` default to 20 and 0.02
     // respectively.
     printSalaryDetails(2500);
 
-    // Calls the function by passing only the `baseSalary` and `annualIncrement`.
-    // Passes the defaultable parameter as a key-value pair when invoking the
-    // function. The `bonusRate` defaults to 0.02.
+    // Call the function by passing only the `baseSalary` and `annualIncrement`.
+    // `annualIncrement` is passed as a named argument.
```

Comment:
```suggestion
    // The `annualIncrement` is passed as a named argument.
```

Reply:
We don't need "the" here right? `annualIncrement` is the name of the parameter.

Same applies for the other comments too.

Reply:
We need "the" before a noun. Particularly, we need it here because it's the start of the sentence.

Reply:
How about if the variable name was `a`.

Would it make sense to say
"// The `a` is passed as a named argument."

IMO, it doesn't sound right. It's just an identifier. Thoughts?

Reply:
It's not simply an "a" in this context. The parameter name is meaningful and that is why we need "the".

Reply:
So in examples, we've used backticks to emphasize keywords and variable names. Because these variables are rendered differently, IMO we should adopt a consistent approach irrespective of the variable names. 

Even though this is the start of a sentence, since it is rendered differently, IMO, it does have the required differentiation.

Reply:
@praneesha I've changed the comments a bit with https://github.com/ballerina-platform/ballerina-lang/commit/6e1558f99d7e1e98fd685e2ef621fea4a0f4f683. :) Can you please check?

Reply:
+1 to add the word parameter.

Reply:
Thanks, will be resolving the rest of the suggestions too.